### PR TITLE
Add new setting to change the app language

### PR DESCRIPTION
**Pull request by lm41**
A new App language option has been added in settings to allow users to change the language of the app directly in the SettingsActivity. The Preference leads the user to the Per-App language settings screen of the android system.

This is a great addition because the language setting is now accessable from the app settings.

### CODE REVIEW

**Comment by affederaffe** [comment]
LGTM

File: app/src/main/java/com/sapuseven/untis/activities/SettingsActivity.kt [hunk 1]
```diff
@@ -255,6 +257,25 @@ class SettingsActivity : BaseComposeActivity() {
 												dataStore = dataStorePreferences.flingEnable
 											)
 										}
+										if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+											PreferenceCategory(stringResource(id = R.string.preference_app_language)) {
```
Comment (SapuSeven) [inline app/src/main/java/com/sapuseven/untis/activities/SettingsActivity.kt]: Please use a separate string resource for the category.
1. In English, categories are capitalized while settings are lowercase ("App Language" vs "App language")
2. We might want to add other interface settings in the future, so they should be independent

**Review by SapuSeven — CHANGES_REQUESTED** [review]
Looks good otherwise

**Review by SapuSeven — APPROVED** [review]

**Comment by SapuSeven** [comment]
Thanks!

**Pull request merged**